src/Rings/solving.jl

```diff
@@ -180,7 +181,30 @@ function rational_solutions(I::MPolyIdeal{<:MPolyRingElem})
   end
   #for technical reasons (evaluation) the points are actually at this
   #point constant polynomials, hence:
-  return [[constant_coefficient(x) for x in r] for r in rts]
+  return Vector{elem_type(K)}[elem_type(K)[constant_coefficient(x) for x in r] for r in rts]
+end
+
+@doc raw"""
+    rational_solutions(K::Field, I::MPolyIdeal) -> Vector{Vector}
+
+Given a zero-dimensional ideal or homogenous one-dimension ideal, return all
+rational elements of the vanishing set over the field $K$.
+
```

Comment:
As this starts the documentation on solving, I suggest to add more information here. What is `K` in relation to the field over which `I` is defined? And, is the latter supposed to be QQ? Should these things be asserted in the code? Should we embed this into adding other things from `solving.jl` together with a short introducction? @ederc?